tasks/src/main/scala/dagr/tasks/vc/VarDictJava.scala

```diff
@@ -150,20 +154,21 @@ class VarDictJavaEndToEnd
     val tmpVcf = f(".vcf")
 
     val vardict = new VarDictJava(
-      tumorBam          = tumorBam,
-      bed               = bed,
-      ref               = ref,
-      tumorName         = tn,
-      minimumAf         = minimumAf,
-      includeNonPf      = includeNonPf,
-      minimumQuality    = minimumQuality,
-      maximumMismatches = maximumMismatches,
-      minimumAltReads   = minimumAltReads,
-      pileupMode        = pileupMode,
-      minThreads        = minThreads,
-      maxThreads        = maxThreads
+      tumorBam            = tumorBam,
+      bed                 = bed,
+      ref                 = ref,
+      tumorName           = tn,
+      minimumAf           = minimumAf,
+      includeNonPf        = includeNonPf,
+      minimumQuality      = minimumQuality,
+      maximumMismatches   = maximumMismatches,
+      minimumAltReads     = minimumAltReads,
+      pileupMode          = pileupMode,
```

Comment:
Should we either add validation that `if (allSites) require(pileup, "...")` or have this line be `pileupMode = pileupMode || allSites`?